### ISSUE TITLE
Should we add the Lewanski review to "publications"?

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/tskit-site#231]
This is the intro to ARGs: https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1011110. It's not from the tskit "stable", but it does have an entire box (Box 3) about tree sequences. What are the qualifications for putting a paper onto https://tskit.dev/learn/?filter=paper#tutorials-publications-videos ?

**Comment by jeromekelleher** [comment]
If we put up papers from people not directly from the core tskit-dev community we should ask them first.

**Comment by hyanwong** [comment]
Very good point. For something that's general like the Lewanski review, we could say "Although not written by anyone from the core Tskit-dev community, this gives a good general review of ...."?

**Comment by jeromekelleher** [comment]
Sure, still polite to ask though.

**Comment by hyanwong** [comment]
> Sure, still polite to ask though.

Yes, definitely. I'm happy to ask them. Do we actually want a few publications like that on the tskit.dev site, though? I think in this case, perhaps (because it is a useful ref for people wanting to know about _tskit_), but not more generally?

**Comment by jeromekelleher** [comment]
I don't really know, it's a sensitive point. I would default to not listing people's papers along with our own in.

**Comment by hyanwong** [comment]
I agree, definitely that should be the default (and less work for us). I'm simply wondering if this should be the exception?

**Comment by jeromekelleher** [comment]
Let's ask them?

**Comment by hyanwong** [comment]
Gideon Bradburd was very happy for this to go up, so in this (exceptional) case, I think we probably should.
